platform/linux-generic/pool/subsystem.c

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2017, ARM Limited. All rights reserved.
+ *
+ * Copyright (c) 2017, Linaro Limited
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+#include <odp_module.h>
+
+#define SUBSYSTEM_VERSION 0x00010000UL
```

Comment:
This version info does mean per each subsystem, in future the usage could be that modules who impl this subsystem can check the version compatibility, like, for example:

NXP module impls "pktio" subsystem version 1.0, but the library contains "pktio" subsystem version 2.0.

For versioning there can be a large feature design, currently only this early support to be able to specify a version number for each subsystem in definition.